experimental/dds/tree2/src/core/edit-manager/editManager.ts

```diff
@@ -218,7 +226,7 @@ export class EditManager<
 		return (this.getLastCommit() ?? fail("No sequenced changes")).change;
 	}
 
-	public getLastCommit(): Commit<TChangeset> | undefined {
+	public getLastCommit(): GraphCommit<TChangeset> | undefined {
```

Comment:
Now that this function is being used outside of this class and you've upgrade the return type, want to make it not have an ambiguous name too? `getTrunkHead` would be good, probably. And it doesn't return `undefined` ever, so that can go away, and the one call site just above in `getLastSequencedChanged` can get rid of the fail check!